action.yml

```diff
@@ -46,6 +46,7 @@ runs:
     - name: Relink Docker client, Docker Compose, and Colima
       if: ${{ steps.homebrew-cache.outputs.cache-hit == 'true' }}
       run: |
+        brew install qemu
```

Comment:
I actually think it's better to install all the missing dependencies of Colima. Can this be done with `brew install --only-dependencies colima`.

Reply:
Then caching Lima is useless. How about `brew install -only-dependencies lima`

Reply:
Caching Lima is not useless. Brew will only install what's missing, so the cache of everything else is still useful. 

For example, I have Lima installed and I ran `brew install lima`: 

```
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 2 taps (robusta-dev/krr and homebrew/cask).
==> New Casks
applite

You have 2 outdated formulae installed.

Warning: lima 0.17.0 is already installed and up-to-date.
To reinstall 0.17.0, run:
  brew reinstall lima
```

So I got only a warning, nothing was reinstalled because that's not what I asked `brew` to do.

Reply:
Also it makes sense to trigger the install of dependencies start from the topmost package.